src/rgw/driver/rados/rgw_notify.cc

```diff
@@ -381,23 +390,31 @@ class Manager : public DoutPrefixProvider {
 
         entries_persistency_tracker& notifs_persistency_tracker = topics_persistency_tracker[queue_name];
         spawn::spawn(yield, [this, &notifs_persistency_tracker, &queue_name, entry_idx, total_entries, &end_marker,
-                             &remove_entries, &has_error, &waiter, &entry](yield_context yield) {
+                             &remove_entries, &has_error, &waiter, &entry, &tenant_name, &migration_in_process](yield_context yield) {
             const auto token = waiter.make_token();
             auto& persistency_tracker = notifs_persistency_tracker[entry.marker];
-            auto result = process_entry(this->get_cct()->_conf, persistency_tracker, entry, yield);
-            if (result == EntryProcessingResult::Successful || result == EntryProcessingResult::Expired) {
-              ldpp_dout(this, 20) << "INFO: processing of entry: " << 
-                entry.marker << " (" << entry_idx << "/" << total_entries << ") from: " << queue_name
-                << (result == EntryProcessingResult::Successful? " ok": " expired") << dendl;
+            auto result = process_entry(this->get_cct()->_conf, persistency_tracker, entry, tenant_name, yield);
+            if (result == EntryProcessingResult::Successful || result == EntryProcessingResult::Expired
+                || result == EntryProcessingResult::Migrating) {
+              std::string return_status = "ok";
```

Comment:
if `return_status` is only used in the debug log, please avoid setting it here.
in line 408 you can just log the enum, or convert the enum to string there, so it is not calculated if debug level is < 20

Reply:
done!